gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -215,13 +216,26 @@ def reset(self):
         self.get_possible_actions(self.turn)
 
     def render(self, mode='human'):
+        """
+        Render current game state with PyGame
+        """
         if self.game is None:
             self.game = XiangQiGame()
             self.game.on_init(self.agent_piece, self.enemy_piece)
         self.game.render()
 
     def close(self):
-        pass
+        """
+        Free up resources and gracefully exit
+        """
+        self.game.cleanup()
+
+    def seed(self, seed=None):
```

Comment:
Hmm what is this method actually for? I guess there's pretty much no randomness in XiangQi right? Hmm

Reply:
I am not 100% sure about the use case of `seed()`. All I know is they are used to save the seed value that is responsible for producing randomness of an environment, and users can go back and use this seed value to reproduce the same environment behavior for experiment purposes. So I think you have the correct understanding. Hmm yeah I was not sure if I can leave it as not implemented.